frontend/packages/process-editor/src/types/SelectionChangeEvent.ts

```diff
@@ -0,0 +1,16 @@
+import { type BpmnBusinessObjectEditor } from './BpmnBusinessObjectEditor';
+
+export type SelectionChangedEvent = {
+  newSelection: Array<{
+    element: {
+      id: string;
+      businessObject: BpmnBusinessObjectEditor;
+    };
+  }>;
+  oldSelection: Array<{
```

Comment:
I see that the bpmnIo forum uses this as well, but since we are not actively using it we might not need it? (yet?) 🤷‍♀️

Reply:
I personally think this is a nice thing to keep. 
Since it's a type definition and not something we maintain this could indicate to people working on this code later that there is an oldSelection property on the event. Especially since the bpmn-js/diagram-js docs are lacking a comprehensive description of the events. 
Do you agree?

Reply:
Yes, I was thinking the same.